src/assets.js

```diff
@@ -0,0 +1,31 @@
+import { eth, dai, xdai, ERC20Asset } from '@burner-wallet/assets';
+import burnerlogo from './burnerwallet.png';
+import bufficorn from './bufficorn.png';
+
+export const assets = [xdai, dai, eth];
+export let token = null;
+export let WEB3_PROVIDER = 'https://dai.poa.network';
+export let ERC20IMAGE;
+export let LOADERIMAGE = burnerlogo;
+export let XDAI_PROVIDER = 'https://dai.poa.network'
```

Comment:
Just an FYI: In the LeapDAO repo we've gotten rid of `WEB3_PROVIDER` and `XDAI_PROVIDER`. Here they're the same value, so technically they should be a single variable too.

Reply:
Eventually all the provider code will be removed, as it is already handled by burner-core, so I'm not sure if it makes sense to integrate that part of the config file here.

Reply:
OK